src/api/app/models/bs_request.rb

```diff
@@ -352,6 +352,10 @@ def superseding
     BsRequest.where(superseded_by: number)
   end
 
+  def first_action_type
```

Comment:
I understand you strive for consistency. But have you considered what this means for developers? I have to look up what `BsRequest.first_action_type` does in the code. I **do not** have to look up what `BsRequest.bs_request_actions.first.action_type` does.